jax/_src/numpy/lax_numpy.py

```diff
@@ -7443,9 +7467,40 @@ def cross(a, b, axisa: int = -1, axisb: int = -1, axisc: int = -1,
   return moveaxis(c, 0, axisc)
 
 
-@util.implements(np.kron)
 @jit
 def kron(a: ArrayLike, b: ArrayLike) -> Array:
+  """Compute the Kronecker product of two input arrays.
+
+  JAX implementation of :func:`numpy.kron`.
+
+  The Kronecker product is an operation on two matrices of arbitrary size that
+  produces a block matrix. Each element of the first matrix ``a`` is multiplied by
+  the entire second matrix ``b``. If ``a`` has shape (m, n) and ``b`` has shape (p, q),
+  the resulting matrix will have shape (m * p, n * q). The blocks of the resulting
+  matrix are formed by multiplying each element of ``a`` by the entire matrix ``b``.
```

Comment:
This statement is repeated twice; I'd remove this second one.

Reply:
done.